be/src/olap/delta_writer_v2.cpp

```diff
@@ -221,8 +223,10 @@ void DeltaWriterV2::_build_current_tablet_schema(int64_t index_id,
 
     _tablet_schema->set_table_id(table_schema_param->table_id());
     // set partial update columns info
-    _tablet_schema->set_partial_update_info(table_schema_param->is_partial_update(),
-                                            table_schema_param->partial_update_input_columns());
+    _partial_update_info = std::make_shared<PartialUpdateInfo>();
+    _partial_update_info->init(*_tablet_schema, table_schema_param->is_partial_update(),
```

Comment:
It's better to use a `shared_ptr<TabletSchema>` as it's first parameter, rather than a const reference. Can you guarentee that the life cycle of `_tablet_schema` will be longger than `_partial_update_info`, even after several times copy of `_partial_update_info`?